react/components/EXPERIMENTAL_Table/index.tsx

```diff
@@ -72,13 +72,19 @@ export const measuresPropTypes = {
 }
 
 export const tablePropTypes = {
+  children: PropTypes.node,
```

Comment:
New props?

Reply:
no, they just weren't defined in the propTypes